### PR TITLE
fix(image): fix loading proper size image

### PR DESCRIPTION
**Pull request by kevinlee11**
## Describe the problem this PR addresses
Fixes images being loaded at full size instead of the corresponding sizes attribute passed in.

## Describe the changes in this PR
Add the sizes attribute when creating the image element for loading.

### CODE REVIEW

**Review by sohnj877 — APPROVED** [review]
Thanks!

File: src/components/Image/src/Image.vue [hunk 1]
```diff
@@ -171,6 +171,12 @@ export default {
 				img.srcset = this.srcset;
 			}
 
+			// Needed to not load the full size image
+			// and match the size loaded in the UI
+			if (this.$attrs?.sizes) {
```
Comment (pretzelhammer) [inline src/components/Image/src/Image.vue:176]: i feel like we probably should just make `sizes` a prop if we're expecting it and using it inside the component

**Comment by landondurnan** [comment]
this is already merged so it doesn't really matter, but this should have been considered a feature rather than a bug fix. While the feature is fixing a bug in the consuming app, Maker didn't have the feature before and this is an addition rather than a fix.

**Comment by kevinlee11** [comment]
So Maker did have this feature before, it was just being passed implicitly through `v-bind="$attrs"`

**Pull request merged**